Update Configuring Custom Pages (Error and Challenge).md (#10637)
cloudflare/cloudflare-docs: content/support/More Dashboard Apps/Cloudflare Custom Pages/Configuring Custom Pages (Error and Challenge).md

tags.
{{</Aside>}}

{{<Aside type="warning">}}

> Please make sure that the meta tag `referrer` is not present in the HTML code your custom error page as it might disrupt the Cloudflare Challenge:

Reviewer: ```suggestion
Please make sure the meta tag `referrer` is not present in your custom error page's HTML code as it might disrupt the Cloudflare Challenge:
```

> {{<Aside type="warning">}}
> Please make sure that the meta tag `referrer` is not present in the HTML code your custom error page as it might disrupt the Cloudflare Challenge:
> `meta name="referrer"`
> {{</Aside>}}

Reviewer 2: ```suggestion
{{<Aside type="warning">}}
Make sure that the `referrer` meta tag is not present in the HTML code of your custom error page since it might disrupt the Cloudflare Challenge:
`<meta name="referrer" (...) />`
{{</Aside>}}
```

Reviewer: Maybe we should replace `might disrupt` with `will disrupt` too or make it clear the challenge page won't work at all.
